src/library/scala/Unit.scala

```diff
@@ -28,6 +28,7 @@ final abstract class Unit private extends AnyVal {
 }
 
 object Unit extends AnyValCompanion {
+  def void[A](x: A): Unit = ()
```

Comment:
`point` or `unit`

Reply:
`x` is still not been used here.

Reply:
That's right, that's the reason this method exists.